src/Materials/Textures/babylon.renderTargetTexture.ts

```diff
@@ -49,7 +49,7 @@
         */
         public onAfterUnbindObservable = new Observable<RenderTargetTexture>();
 
-        private _onAfterUnbindObserver: Observer<RenderTargetTexture>;
+        protected _onAfterUnbindObserver: Observer<RenderTargetTexture>;

```

Comment:
you're right, I used to and now I don't. I will double check what I changed to protected